feat: Test cases for Scheduled Message feature (#183)
mattermost/mattermost-test-management: data/test-cases/channels/schedule-messages/4-Failed-Schedule-Message.md

### Scenario 5: Verify LHS Count displayed when scheduled message fails to be sent.

1. Log into User's account.
2. Verify there is no error icon near Drafts on the LHS
3. Leave the channel where a message was scheduled to be sent.
4. Verify the user can see Error icon in red color.
5. Hover on the Icon a tooltip saying `One of your scheduled drafts cannot be sent`.
6. Goto Scheduled Tab in Drafts page.
7. Hoven on the Failed message and delete it.
8. Verify the Error message is not shown on the LHS anymore.

### Scenario 6: Verify Schedule message fails if the root post is moved to Private channel on the same team.

1. Log into User A's account and create a Root post in a channel.
2. Reply to the root post with some messages.
3. Log into User B's account and create a threaded schedule message to the root post.
4. From User A's account, Using **Move Threads** feature, move the root post to a private Channel where User B is not a member.
5. Log into User B's account.
6. Goto Scheduled messages tab.
7. Verify the message has a tag `Not in the channel` in red alert.
8. Verify the User can see `No Channel` in the Channels section.

> ### Scenario 7: Verify Schedule message fails if the root post is moved to another team channel.

Reviewer: ```suggestion
### Scenario 7: Verify Schedule message fails if the root post is moved to another team.
```

1. Log into User A's account and create a Root post in a channel.
2. Reply to the root post with some messages.

> 3. Log into User B's account and create a threaded schedule message to the root post.

Reviewer: ```suggestion
3. Log into User B's account and create a threaded scheduled message on the root post.
```

4. From User A's account, Using **Move Threads** feature, move the root post to another Team where User B is not a member.
5. Log into User B's account.

> 6. Goto Scheduled messages tab.

Reviewer: ```suggestion
6. Go to Scheduled messages tab.
```
